website/content/docs/overview/architecture/index.md

```diff
@@ -9,25 +9,57 @@ weight: 10
 The information below is outdated
 {{</ warning >}}
 
-{{< name >}} takes a **modular** approach to provide an open platform for controlling and securing the use of data across an organization. The figure below showcases the current architecture of the {{< name >}} platform, running on top of OpenShift Container Platform. The storage systems shown in the lower half of the figure are merely an example.
```

Comment:
Do we want to include the copy/ingest capabilities we now include?  If so, then it's not just about using data but also about governing the copying of data.  One example of a copy scenario is the initial importing of data into the secured environment.

Reply:
@simanadler I would prefer this to be out of scope for this PR if that's ok